cmd/fleet/main.go

```diff
@@ -430,8 +434,21 @@ func NewFleetServer(cfg *config.Config, bi build.Info, reporter status.Reporter)
 
 type runFunc func(context.Context) error
 
-// Run runs the fleet server.
+// Run runs the fleet server
 func (f *FleetServer) Run(ctx context.Context) error {
+	// Set up a noop consumer for the ready signal
```

Comment:
Trying to understand how this change is related?

Reply:
when adding the limit calculation in response to a config being passed down there was a race condition introduced with one of the integration tests that read off the fleet input. i added the ready channel to the `Run` command so the test could use it, and added this method to interface cleanly with the existing code.